.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
+.vscode/
```

Comment:
I'd suggest you to shift to goland ide

Reply:
Or, setup global gitignore.

Reply:
FWIW whatever IDE one uses it's fine for us.

VSCode is as good (if not better) than Goland, having `.vscode` ignored is okay.